src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main (String[] args) {
+        Machine[] machines = {new Truck(), new Bulldozer(), new Excavator()};
 
+        for (Machine machine: machines) {
+            machine.doWork();
```

Comment:
The messages within the 'doWork' method should be informative and indicate what type of machine is working. Ensure that each 'Machine' subclass has an overridden 'doWork' method with a specific message.